utils.groovy

```diff
@@ -1,25 +1,20 @@
-workdir = env.WORKSPACE
```

Comment:
so now that we aren't setting this any longer we just always default to `env.WORKSPACE` anyway?

Reply:
Yup, exactly!